packages/test-harness/src/config/init.lua

```diff
@@ -0,0 +1,10 @@
+local temp_dir = os.getenv("TEMP_DIR")
+local repo_root = os.getenv("CURSORLESS_REPO_ROOT")
+
+vim.cmd("source " .. temp_dir .. "/BufOnly.vim/plugin/BufOnly.vim")
+
+vim.opt.runtimepath:append(temp_dir .. "/talon.nvim")
+vim.opt.runtimepath:append(repo_root .. "/dist/cursorless.nvim")
```

Comment:
should this be "/cursorless.nvim" now instead of "/dist/cursorless.nvim"? cc @pokey I wasn't sure as you changed the logic recently.

Reply:
I think so? I just symlink the bundled js in during dev now so I think you're right

Reply:
I tested it because and it fails when using `/cursorless.nvim` instead of `/dist/cursorless.nvim`:

```
 Started at 1721643147000
Run xvfb-run -a pnpm -F @cursorless/test-harness testNeovim
> @cursorless/test-harness@0.1.0 testNeovim /home/runner/work/cursorless/cursorless/packages/test-harness
> env CURSORLESS_MODE=test my-ts-node src/scripts/runNeovimTestsCI.ts
cli: /home/runner/nvim-stable/bin/nvim
init.lua copying done
listing nvim/:
	init.lua
nvim started done
pid: 4442
listing cursorless-neovim/out/:
	NeovimCommandServerApi.d.ts
	NeovimCommandServerApi.d.ts.map
	constructTestHelpers.d.ts
	constructTestHelpers.d.ts.map
	extension.d.ts
	extension.d.ts.map
	index.cjs
	index.d.ts
	index.d.ts.map
	nvim_node.log
	registerCommands.d.ts
	registerCommands.d.ts.map
tail neovim test started
waiting for tests to finish ...
neovim test: 2024-07-22 10:12:29 INF handleRequest: 
neovim test: 2024-07-22 10:12:29 INF handleNotification: 
neovim test: 2024-07-22 10:12:29 ERR [cursorless-neovim] Error: Cannot find module '/home/runner/work/cursorless/cursorless/cursorless.nvim//node/cursorless-neovim/'
neovim test: Require stack:
neovim test: - /opt/hostedtoolcache/node/20.12.1/x64/lib/node_modules/neovim/lib/host/factory.js
neovim test: - /opt/hostedtoolcache/node/20.12.1/x64/lib/node_modules/neovim/lib/host/index.js
neovim test: - /opt/hostedtoolcache/node/20.12.1/x64/lib/node_modules/neovim/bin/cli.js
neovim test:     at Module._resolveFilename (node:internal/modules/cjs/loader:1143:15)
neovim test:     at Module._load (node:internal/modules/cjs/loader:984:27)
neovim test:     at Module.require (node:internal/modules/cjs/loader:1231:19)
neovim test:     at require (node:internal/modules/helpers:179:18)
neovim test:     at createPlugin (/opt/hostedtoolcache/node/20.12.1/x64/lib/node_modules/neovim/lib/host/factory.js:47:31)
neovim test:     at loadPlugin (/opt/hostedtoolcache/node/20.12.1/x64/lib/node_modules/neovim/lib/host/factory.js:63:16)
neovim test:     at Host.getPlugin (/opt/hostedtoolcache/node/20.12.1/x64/lib/node_modules/neovim/lib/host/index.js:57:43)
neovim test:     at Host.<anonymous> (/opt/hostedtoolcache/node/20.12.1/x64/lib/node_modules/neovim/lib/host/index.js:88:33)
neovim test:     at Generator.next (<anonymous>)
neovim test:     at /opt/hostedtoolcache/node/20.12.1/x64/lib/node_modules/neovim/lib/host/index.js:31:71
neovim test: 2024-07-22 10:12:29 ERR [cursorless-neovim] Error loading child ChildPlugin /home/runner/work/cursorless/cursorless/cursorless.nvim//node/cursorless-neovim/
neovim test: 2024-07-22 10:12:29 ERR Could not load plugin: /home/runner/work/cursorless/cursorless/cursorless.nvim//node/cursorless-neovim/
neovim test: 2024-07-22 10:12:30 INF handleNotification: 
neovim test: 2024-07-22 10:12:30 ERR [test-harness] Error: Cannot find module '/home/runner/work/cursorless/cursorless/cursorless.nvim//node/test-harness/'
neovim test: Require stack:
neovim test: - /opt/hostedtoolcache/node/20.12.1/x64/lib/node_modules/neovim/lib/host/factory.js
neovim test: - /opt/hostedtoolcache/node/20.12.1/x64/lib/node_modules/neovim/lib/host/index.js
neovim test: - /opt/hostedtoolcache/node/20.12.1/x64/lib/node_modules/neovim/bin/cli.js
neovim test:     at Module._resolveFilename (node:internal/modules/cjs/loader:1143:15)
neovim test:     at Module._load (node:internal/modules/cjs/loader:984:27)
neovim test:     at Module.require (node:internal/modules/cjs/loader:1231:19)
neovim test:     at require (node:internal/modules/helpers:179:18)
neovim test:     at createPlugin (/opt/hostedtoolcache/node/20.12.1/x64/lib/node_modules/neovim/lib/host/factory.js:47:31)
neovim test:     at loadPlugin (/opt/hostedtoolcache/node/20.12.1/x64/lib/node_modules/neovim/lib/host/factory.js:63:16)
neovim test:     at Host.getPlugin (/opt/hostedtoolcache/node/20.12.1/x64/lib/node_modules/neovim/lib/host/index.js:57:43)
neovim test:     at Host.<anonymous> (/opt/hostedtoolcache/node/20.12.1/x64/lib/node_modules/neovim/lib/host/index.js:88:33)
neovim test:     at Generator.next (<anonymous>)
neovim test:     at /opt/hostedtoolcache/node/20.12.1/x64/lib/node_modules/neovim/lib/host/index.js:31:71
neovim test: 2024-07-22 10:12:30 ERR [test-harness] Error loading child ChildPlugin /home/runner/work/cursorless/cursorless/cursorless.nvim//node/test-harness/
neovim test: 2024-07-22 10:12:30 ERR Could not load plugin: /home/runner/work/cursorless/cursorless/cursorless.nvim//node/test-harness/
```

At the end after checking again, you actually decided to use `/dist/cursorless.nvim`, since indeed the symlinks are only created locally. And the only reason to use `cursorless.nvim` is to get symbols loaded which we don't need on CI.